demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2862,6 +2941,14 @@ export class DemoMeetingApp
       return new SegmentationProcessor();
     }
 
+    if (videoFilter === 'Resize (1/1)') {
+      return new ResizeProcessor(1);  // 1/1 Aspect Ratio
+    }
+
+    if (videoFilter === 'Resize (9/16)') {
```

Comment:
Can we just do one? Not sure if we need both.

Reply:
I think these two are good implementations. I do not feel strongly either way though. Let me know if you want me to remove; which one should I remove then?

Reply:
I think its okay to stick with one - after all, this is a demo. We just need to show how its done once.

Reply:
Thanks @michhyun1, I will remove the 1/1 aspect ratio resize processor and keep the 9/16.